### PR TITLE
Rubocop fixes

### PR DESCRIPTION
**Pull request by orangewolf**
This PR pulls in a bunch of Rubocop fixes from HykuUp's code base. Doing this as a separate PR since one gets a little bored looking at the same magic comment in 252 files. The only interesting bits (in my opinion) are the spec/support file changes.  There are two things (one removed file and one removed section in `multitenancy_metadata.rb` that massively sped up our spec runs and did not appear to be doing anything.  See https://github.com/samvera/hyku/pull/1640/commits/fef25caf4b71cce5d2f82bc4cc2e3db340e31772 to look at just that part.

@samvera/hyku-code-reviewers

### CODE REVIEW

File: app/models/endpoint.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 class Endpoint < ApplicationRecord
   has_one :account
+
+  def switchable_options
```
Comment (cjcolvar) [inline app/models/endpoint.rb:6]: Can you explain this change more?  Why is this needed?

File: app/forms/hyrax/forms/admin/appearance.rb [hunk 2]
```diff
@@ -259,6 +259,7 @@ def self.customization_params # rubocop:disable Metrics/MethodLength
         end
 
         def font_import_body_url
+          headline = headline_font.split('|').first.to_s.tr(" ", "+")
```
[the remark this answers is not in the document]
Comment (orangewolf) [reply app/forms/hyrax/forms/admin/appearance.rb:262]: agreed. looks like accidental copy-pasta

File: app/models/endpoint.rb (hunk 1 above)
Comment (orangewolf) [reply app/models/endpoint.rb:6]: That's the problem with these things being "out" on their own branches so long. I vaguely remember this code.  Let's see what past Rob had to say:

```
commit 825029de758cffaa2425674b88e96eaa91ffbb05
Author: Rob Kaufman <rob@notch8.com>
Date:   Thu Feb 21 12:15:07 2019 -0800

    some endpoints have empty options, clearing (for example) the fedora url, which then causes the process to get and stay stuck
```

I think a code comment to that affect is worth while.

Comment (orangewolf) [reply app/models/endpoint.rb:11]: we were seeing that if you had a blank solr or fedora url you could basically cause passenger processes to stop responding. that seems like it was worth fixing and trying to future proof a little.

**Comment by orangewolf** [comment]
Being in a state where more of Hyku is Rubocop compliant makes the jump to current Bixby feel pretty achievable.

File: app/models/endpoint.rb (hunk 1 above)
Comment (cjcolvar) [reply app/models/endpoint.rb:11]: :+1: Thanks!
